pkg/model/sdk_helper.go

```diff
@@ -382,20 +384,27 @@ func (a *SDKAPI) SDKAPIInterfaceTypeName() string {
 }
 
 // Override the operation type and/or resource name if specified in config
-func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) (OpType, string) {
+func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) ([]OpType, string) {
 	opType, resName := GetOpTypeAndResourceNameFromOpID(opID)
-
+	opTypeArray := make([]OpType, 0)
+	// TODO: or use var opTypeArray []OpType
```

Comment:
Please remove the `TODO` comment. I suggest using `opTypes := []OpType{}` here

Reply:
Also the code below will be simpler if you initialize the array using `opTypes := []OpType{opType}`

Reply:
You can get something very similar to 
```go
func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) ([]OpType, string) {
	opType, resName := GetOpTypeAndResourceNameFromOpID(opID)
	opTypes := []OpType{opType}

	if cfg == nil {
		return opTypes, resName
	}
	if operationConfig, exists := cfg.Operations[opID]; exists {
		if operationConfig.ResourceName != "" {
			resName = operationConfig.ResourceName
		}
		for _, operationType := range operationConfig.OperationType {
			opType = OpTypeFromString(operationType)
			opTypes = append(opTypes, opType)
		}
	}
	return opTypes, resName
}

```

Reply:
thanks, this was the intent of the mentioned cleanup :)